code/modules/detective_work/evidence.dm

```diff
@@ -80,3 +80,181 @@
 	else
 		to_chat(user, "[src] is empty.")
 		icon_state = "evidenceobj"
+
+/obj/item/forensics
+	icon = 'icons/obj/forensics/forensics.dmi'
+	w_class = WEIGHT_CLASS_TINY
+
+/obj/item/sample
+	name = "\improper Forensic sample"
+	icon = 'icons/obj/forensics/forensics.dmi'
+	w_class = WEIGHT_CLASS_TINY
+	var/list/evidence = list()
+
+/obj/item/sample/New(newloc, atom/supplied)
+	. = ..()
+	if(supplied)
+		copy_evidence(supplied)
+		name = "[initial(name)] (\the [supplied])"
+
+/obj/item/sample/print/New(newloc, atom/supplied)
+	. = ..()
+	if(length(evidence))
+		icon_state = "fingerprint1"
+
+/obj/item/sample/proc/copy_evidence(atom/supplied)
+	if(length(supplied.suit_fibers))
+		evidence = supplied.suit_fibers.Copy()
+		supplied.suit_fibers.Cut()
```

Comment:
You might want to rename this to `take` or `transfer` evidence -- I feel like `copy` implies that the original isn't changed (though it's cut)